db/dcrpg/pgblockchain.go

```diff
@@ -991,43 +991,13 @@ func (pgb *ChainDB) AgendaVotes(agendaID string, chartType int) (*dbtypes.Agenda
 		return nil, nil
 	}
 
-	votingStartHeight, err := pgb.RCIStartHeight(agendaInfo)
-	if err != nil {
-		return nil, err
-	}
-
 	avc, err := retrieveAgendaVoteChoices(ctx, pgb.db, agendaID, chartType,
-		votingStartHeight, agendaInfo.VotingDone)
+		agendaInfo.VotingStarted, agendaInfo.VotingDone)
 	return avc, pgb.replaceCancelError(err)
 }
 
-// RCIStartHeight returns the startheight of the RCI window when voting for
```

Comment:
```bash
➜  dcrctl git:(master) ./dcrctl getblockchaininfo                           
{
  "chain": "mainnet",
  "blocks": 326880,
  "headers": 326880,
  "syncheight": 326808,
  "bestblockhash": "000000000000000027aefe4bff091cc25562a5f87c2ad83ab1664992f7ebcf04",
  "difficulty": 405549711,
  "verificationprogress": 1,
  "chainwork": "00000000000000000000000000000000000000000002ddc8dbe706daafda8bae",
  "initialblockdownload": false,
  "maxblocksize": 393216,
  "deployments": {
    "fixlnseqlocks": {
      "status": "started",
      "since": 326656,
      "starttime": 1548633600,
      "expiretime": 1580169600
    },
    "lnfeatures": {
      "status": "active",
      "since": 189568,
      "starttime": 1505260800,
      "expiretime": 1536796800
    },
    "lnsupport": {
      "status": "active",
      "since": 149248,
      "starttime": 1493164800,
      "expiretime": 1508976000
    },
    "sdiffalgorithm": {
      "status": "active",
      "since": 149248,
      "starttime": 1493164800,
      "expiretime": 1524700800
    }
  }
}
```

This payload returns all the data we need to calculate RCI StartHeight for any agenda whose vote status is past  or is in the started vote status. Thus `RCIStartHeight(agendaInfo dbtypes.MileStone) (int64, error) {` is not necessary.